src/eggd_mosdepth.sh

```
@@ -11,10 +11,13 @@ main() {
     dx-download-all-inputs
     find ~/in -type f -name "*" -print0 | xargs -0 -I {} mv {} ~/input
 
-    
     # get reference build used for mapping from bam
-    ref=$(samtools view -H input/$bam_prefix.bam | grep @SQ | tail -1 | cut -d$'\t' -f2 | cut -d':' -f2)
-    echo $ref >> out/mosdepth_output/$bam_prefix.reference_build.txt
+    ref=$(samtools view -H input/$alignment_file_prefix*.*am | grep @SQ | tail -1 | cut -d$'\t' -f2 | cut -d':' -f2)
+    echo $ref >> out/mosdepth_output/$alignment_file_prefix.reference_build.txt
+
+    # set up reference fasta
+    gunzip input/${reference_fasta_prefix}.fa.gz
```

Shellcheck (0.10.0)

[warning] reference_fasta_prefix is referenced but not assigned.

(SC2154)